src/Datadog.Trace/Util/DomainMetadata.cs

```diff
@@ -100,12 +101,15 @@ public static int AppDomainId
 
         public static bool ShouldAvoidAppDomain()
         {
-            if (AppDomainName.IndexOf("ApplicationInsights", StringComparison.OrdinalIgnoreCase) >= 0)
+            var appDomain = AppDomain.CurrentDomain;
```

Comment:
Please not the Slack discussion on using VAR. Depending on how that goes, this would be an example where var should not be used. :)

Reply:
I really wish, for the sake of my muscle memory, that we'll end-up allowing both :P